src/CalcManager/Ratpack/basex.cpp

```diff
@@ -126,28 +126,28 @@ void _mulnumx( PNUMBER *pa, PNUMBER b )
                     c->cdigit++;
                 }
             }
-            
+
             // If result is nonzero, or while result of carry is nonzero...
             while ( mcy || cy )
             {
-                
+
                 // update carry from addition(s) and multiply.
                 cy += (TWO_MANTTYPE)ptrc[icdigit]+((DWORD)mcy&((DWORD)~BASEX));
-                
-                // update result digit from 
+
+                // update result digit from
                 ptrc[icdigit++]=(MANTTYPE)((DWORD)cy&((DWORD)~BASEX));
-                
+
                 // update carries from
                 mcy >>= BASEXPWR;
                 cy >>= BASEXPWR;
             }
-            
+
             ptrb++;
             ptrc++;
-            
```

Comment:
It looks like you're introducing some trailing whitespace.  I may have missed others... you may want to re-review,

Reply:
There's no whitespace here, just an empty line. I can't find any introduced whitespace.

Reply:
Weird.  I've confirmed you're right by looking at the raw file in your branch.  Not sure why my diffing tool was showing the spaces on this line (and on a couple, otherwise blank, lines as well).